rules/good-email-subject/rule.md

```diff
@@ -21,70 +23,81 @@ guid: 5f1e85d7-f385-4d3b-8338-d49fe11eab91
 
 ---
 
-Just as we should not 'judge a book by its cover' - we will not judge an email by its subject. But, we do! Because users get so many emails, getting your clients and suppliers to take notice of yours in their inbox can be quite a struggle. 
+![Email subjects](https://github.com/user-attachments/assets/9b67be39-50ee-4563-ab4f-0e273613788c)
+
+If there's anything to be learned from spammers, they know how to get your attention. Spammers use very tabloid based, or headline-grabbing subjects, to try and coerce you to open that email. But don't make your email subjects tabloid-tacky, instead follow a good broadsheet paper's style of attention-grabbing lines.
 
 <!--endintro-->
 
+Just as we are told, 'Don't judge a book by its cover', we sometimes judge an email by its subject.
+We send and recieve so many emails each day, therefore its important to use the email Subject to grab your recipient's attention.
+Choosing the right subject can give an email a sense of urgency or importance and allows the user to quickly glance through the inbox to assist with prioritization.
```

Comment:
Hi @Levijj22 

- [ ] 1. Consider starting a new paragraph for this sentence

Reply:
✅ Done

Just as we are told, 'Don't judge a book by its cover', we sometimes judge an email by its subject.
We send and receive so many emails each day, therefore it's important to use the email subject to grab your recipient's attention.

Choosing the right subject can give an email a sense of urgency or importance and allows the user to quickly glance through the inbox to assist with prioritization.
This is especially important when trying to get your clients and suppliers to take notice of your email.